daemon/control.h

```diff
@@ -68,6 +72,11 @@ typedef enum {
 	CONTROL_RESPONSE_CMD_UNSUPPORTED,
 } control_message_t;
 
+/**
+ * Enum represents different commands to control a container
+ */
+typedef enum { CONTAINER_CTRL_START, CONTAINER_CTRL_STOP } container_ctrl_t;
+
```

Comment:
define this in cmld.h as cmld_container_ctrl_t and rename values accordingly.

Reply:
Done (required changing order of includes in some files)